src/clr-angular/utils/_theme.dark.clarity.scss

```diff
@@ -45,74 +45,76 @@
 $clr-global-app-background: #1b2a32;
 
 /*********
- * $clr-global-selection-color
- * Components that have a selected (or active) state use this to change the background-color when an item or element
- * is in the active state.
- * Usages:
- * - _layout.clarity.scss
- * - _datagrid.clarity.scss
- * - _sidenav.clarity.scss
- *
- * Component's with variables affected in the dark theme (this file)
- * - Stack View
- * - Tree View
- * - Dropdown
- */
+  * $clr-global-selection-color
+  * Components that have a selected (or active) state use this to change the background-color when an item or element
+  * is in the active state.
+  * Usages:
+  * - _layout.clarity.scss
+  * - _datagrid.clarity.scss
+  * - _sidenav.clarity.scss
+  *
+  * Component's with variables affected in the dark theme (this file)
+  * - Stack View
+  * - Tree View
+  * - Dropdown
+  */
 $clr-global-selection-color: #324f61;
 
 /*********
- * $clr-global-hover-bg-color
- * Components or elements that can be hovered will have this color applied to their background-color.
- * Component's with variables affected in the dark theme (this file)
- * - Dropdown
- * - Tree View
- */
+  * $clr-global-hover-bg-color
+  * Components or elements that can be hovered will have this color applied to their background-color.
+  * Component's with variables affected in the dark theme (this file)
+  * - Dropdown
+  * - Tree View
+  */
 $clr-global-hover-bg-color: #29414e;
 
 /*********
- * The close class is a utility class that can be applied to elements with a clr-icon thats used for closing something.
- * There are two variables that control the normal and hovered states.
- * Usages:
- * - _close.clarity.scss
- *
- * Component's with variables affected in the dark theme (this file)
- * - ??
- */
-$close-color--normal: #c3ced4;
-$close-color--hover: $clr-white;
+  * The close class is a utility class that can be applied to elements with a clr-icon thats used for closing something.
+  * There are two variables that control the normal and hovered states.
+  * Usages:
+  * - _close.clarity.scss
+  *
+  * Component's with variables affected in the dark theme (this file)
+  * - ??
+  */
+$clr-close-color--normal: #c3ced4;
```

Comment:
I left two other comments re: `--` it looks like we are using them for the _modifier_ part of the variable? If thats what wer are doing here, my preference is to still use one `-`. Since we don't have a strict BEM implementation especially wrt to the `-`'s.